evennia/contrib/grid/ingame_map_display/README.md

```diff
@@ -0,0 +1,48 @@
+# Basic Map
+
+Contribution - helpme 2022
+
+This adds a `map` to a given room which can be viewed with the `map` command. You can
```

Comment:
#It would be relevant to mention already up here that the map is dynamically generated on the fly from the existing grid and that it supports all compass directions + up/down. This is the key defining feature of this compared to other map contribs.